packages/ui-library/src/components/radio/index.stories.ts

```diff
@@ -2,6 +2,10 @@ import { BlrRadioType } from './index';
 import { BlrRadioRenderFunction } from './renderFunction';
 import { html } from 'lit-html';
 import { action } from '@storybook/addon-actions';
+import '../../index';
```

Comment:
You have to remove these lines. The app won't start with these duplicates

Reply:
done